recipe/meta.yaml

```diff
@@ -27,7 +26,7 @@ requirements:
         - setuptools
         - numpy x.x
         - hdf4  # [not win]
-        - hdf5
+        - hdf5 1.8.15.1
```

Comment:
This will break envs that installs both `netcdf` and `hdf5` together. That is fine for `gdal2` though, since I am pinning most packages that results in that combination to `gdal 1.11.4`.

We will be free of this issue once https://github.com/conda-forge/staged-recipes/pull/213 is merged.